articles/azure/azs-gs.md

```diff
@@ -103,15 +103,13 @@ After providing the above details, you should receive a welcome email shortly fr
 
 ***
 
-## Logging in to the Microsoft Azure Stack portal
+## Logging in to the UKCloud for Microsoft Azure portal
```

Comment:
I think elsewhere in the doc we call this the UKCloud Azure Stack portal, so maybe change to that instead? Also, if you change what we call it here, you need to change it in the rest of the article as well.

Reply:
We call it:
- Azure Stack portal
- UKCloud for Microsoft Azure portal
- UKCloud Azure Stack portal
- Microsoft Azure Stack Portal

I 100% agree that we should keep it consistent across all of our docs, but I'm not sure which is the best option. I'm leaning towards "Azure Stack portal" or "UKCloud Azure Stack portal" though, thoughts?

Reply:
I'd suggest we go with UKCloud Azure Stack portal to identify it as specific to us. That's pretty cumbersome, so what we could do is use that full name for the first use in each article and then just use Azure Stack portal for the rest.

Reply:
Have made changes to all mentions of the portal. It's a mixture between UKCloud Azure Stack portal and Azure Stack portal, using the first for headers, the first mentions of the portal in docs and for overview docs (Getting started, FAQs, et cetera.). I used the latter for everywhere else

Reply:
Good approach, thanks